### ISSUE TITLE
worker stream @rehearsal/cli#migrate

### ISSUE THREAD

**Issue by lynchbomb** [rehearsal-js/rehearsal-js#489]
With @rehearsal/cli#migrate during the [task](https://github.com/rehearsal-js/rehearsal-js/blob/a30fb52255febf45bd8be14fb0194288277b60d4/packages/cli/src/commands/migrate.ts#L218-L271) `Converting JS files to TS` in the use-case of migrating a robust application; this task gives the appearance that it has stalled out (it hasn't). 

Instead of a static title for this task can we stream the current work being logged within this task as the title? That way it's obvious work is happening with a bit of transparency around the work being done within this task.

**Comment by egorio** [comment]
There are Winston logger can be configured to output info. I was using --verbose to see progress, but I also thought if we can check the progress of migrate / upgrade somehow from the commander

**Comment by haochuan** [comment]
Yeah I need to update the logger stuff. I think the current `-v` is showing too much stuff, and we need to short version of that

**Comment by haochuan** [comment]
Probably need to just show: which file is being processed one by one in order

**Comment by haochuan** [comment]
Done in https://github.com/rehearsal-js/rehearsal-js/pull/502
